mlflow/tracing/utils.py

```diff
@@ -80,3 +80,31 @@ def format_trace_id(trace_id: int) -> str:
     Format the given integer trace ID to a hex string.
     """
     return f"0x{trace_api.format_span_id(trace_id)}"
```

Comment:
Can we call this `encode` instead of `format`? To be consistent with the `decode_xx` function. BTW can we unify `format_span_id` and `format_trace_id`?

Reply:
> Can we call this encode instead of format?

Sure!

> BTW can we unify format_span_id and format_trace_id?

No, they should be different but I put the wrong function (see above comment).